tests/contrib/django/test_django.py

```diff
@@ -1333,6 +1335,25 @@ def test_middleware_trace_request_ot(client, test_spans, tracer):
     assert sp_request.get_tag("http.method") == "GET"
 
 
+def test_set_request_tags_handles_improperly_configured_error(client):
+    """
+    Since it's difficult to reproduce the ImproperlyConfigured error via django (server setup), will instead
+    mimic the failure by mocking the user_is_authenticated to raise an error.
+    """
+    request = _HttpRequest()
+    request.user = mock.Mock()
+    span = mock.Mock()
+    if django.VERSION >= (1, 10, 1):
+        type(request.user).is_authenticated = mock.PropertyMock(side_effect=django.core.exceptions.ImproperlyConfigured)
+    else:
+        request.user.is_authenticated = mock.Mock(side_effect=django.core.exceptions.ImproperlyConfigured)
+
+    try:
+        _set_request_tags(django, span, request)
```

Comment:
```suggestion
    with pytest.raises(django.core.exceptions.ImproperlyConfigured
):
        _set_request_tags(django, span, request)
```

Reply:
Is it possible to mock the error and have the request go through our instrumentation still? It'd be good to ensure that we handle the error in the integration.